meta/main.yml

```diff
@@ -5,15 +5,19 @@ galaxy_info:
 
   license: CC-BY-SA
 
-  min_ansible_version: 2.2
+  min_ansible_version: 2.4
 
   platforms:
     - name: Ubuntu
       versions:
+        - xenial
         - yakkety
+        - zesty
+        - artful
     - name: Debian
       versions:
         - jessie
+        - stretch
```

Comment:
Are you sure about Debian being here?

Reply:
Yeah, it looks like it is actually there.

Reply:
I'd just get rid of it. It's not tested on Debian, so we shouldn't claim to support it

Reply:
Okay. Your're right, let's leave it at the well-tested ubuntu support.